### PR TITLE
Update activate-using-active-directory-based-activation-client.md

### PR DESCRIPTION
**Pull request by Karl-WE**
Adding Office to the supported list, as this greatly unknown based on my daily expierence as a consultant. 
Too many still use KMS.

### CODE REVIEW

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 1]
```diff
@@ -97,6 +100,12 @@ When a reactivation event occurs, the client queries AD DS for the activation o
 1. Activate your KMS host key by phone or online (Figure 15).
 
     ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
+    
+    > [!NOTE]
+    > For activating a KMS Host Key (CSVLK) for Microsoft Office you need to install the version specific Office Volume license pack on the server where you have installed the Volume Activation Server Role.
```
Comment (illfated) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:105]: Grammar improvement & capitalization in "Volume License Pack" (as shown on https://www.microsoft.com/en-us/download/details.aspx?id=57342 & https://www.microsoft.com/en-us/download/details.aspx?id=49164)
```suggestion
    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 2]
```diff
@@ -115,6 +124,7 @@ To verify your Active Directory-based activation configuration, complete the fol
 
     > [!NOTE]
     > If you are using both KMS and Active Directory-based activation, it may be difficult to see whether a client has been activated by KMS or by Active Directory-based activation. Consider disabling KMS during the test, or make sure that you are using a client computer that has not already been activated by KMS. The **slmgr.vbs /dlv** command also indicates whether KMS has been used.
+> If you would like to see or mass change this, please consider to use [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).
```
Comment (illfated) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:127]:
```suggestion
> To manage individual activations or apply multiple (mass) activations, please consider using [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).
```

Comment (illfated) [reply windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:127]: The line above could possibly be phrased better, but I don't have enough personal experience in this topic to provide a better content for this line.

**Comment by Karl-WE** [comment]
Thank you @illfated!

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 3]
```diff
@@ -97,6 +100,13 @@ When a reactivation event occurs, the client queries AD DS for the activation o
 1. Activate your KMS host key by phone or online (Figure 15).
 
     ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
+    
+    > [!NOTE]
+    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:105]:
```suggestion
    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version-specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 4]
```diff
@@ -97,6 +100,13 @@ When a reactivation event occurs, the client queries AD DS for the activation o
 1. Activate your KMS host key by phone or online (Figure 15).
 
     ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
+    
+    > [!NOTE]
+    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.
+
+    - [Office 2013 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=35584)
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:107]:
```suggestion
    - [Office 2013 VL pack](https://www.microsoft.com/download/details.aspx?id=35584)

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 5]
```diff
@@ -97,6 +100,13 @@ When a reactivation event occurs, the client queries AD DS for the activation o
 1. Activate your KMS host key by phone or online (Figure 15).
 
     ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
+    
+    > [!NOTE]
+    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.
+
+    - [Office 2013 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=35584)
+    - [Office 2016 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=49164)
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:108]:
```suggestion
    - [Office 2016 VL pack](https://www.microsoft.com/download/details.aspx?id=49164)

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 6]
```diff
@@ -97,6 +100,13 @@ When a reactivation event occurs, the client queries AD DS for the activation o
 1. Activate your KMS host key by phone or online (Figure 15).
 
     ![Entering your KMS host key](../images/volumeactivationforwindows81-14.jpg)
+    
+    > [!NOTE]
+    > To activate a KMS Host Key (CSVLK) for Microsoft Office, you need to install the version specific Office Volume License Pack on the server where the Volume Activation Server Role is installed.
+
+    - [Office 2013 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=35584)
+    - [Office 2016 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=49164)
+    - [Office 2019 VL pack](https://www.microsoft.com/en-us/download/details.aspx?id=57342)
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:109]:
```suggestion
    - [Office 2019 VL pack](https://www.microsoft.com/download/details.aspx?id=57342)

```

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md [hunk 7]
```diff
@@ -115,6 +125,7 @@ To verify your Active Directory-based activation configuration, complete the fol
 
     > [!NOTE]
     > If you are using both KMS and Active Directory-based activation, it may be difficult to see whether a client has been activated by KMS or by Active Directory-based activation. Consider disabling KMS during the test, or make sure that you are using a client computer that has not already been activated by KMS. The **slmgr.vbs /dlv** command also indicates whether KMS has been used.
+> To manage individual activations or apply multiple (mass) activations, please consider using [VAMT](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/volume-activation-management-tool).
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:128]:
```suggestion
> To manage individual activations or apply multiple (mass) activations, please consider using the [VAMT](https://docs.microsoft.com/windows/deployment/volume-activation/volume-activation-management-tool).

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove locales as indicated to facilitate localization.

**Comment by Karl-WE** [comment]
Hi @JohanFreelancer9 thanks for your corrections. Much appreciated!

File: windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md (hunk 6 above)
Comment (illfated) [reply windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md:112]: Looks like the suggestion above added a not needed blank line. Going to try suggesting the link change without adding that extra blank line.
```suggestion
    - [Office 2019 VL pack](https://www.microsoft.com/download/details.aspx?id=57342)
```

**Comment by Karl-WE** [comment]
With your review and permission I think this is ready for signoff. Thank you very much! @greg-lindsay

**Review by illfated — APPROVED** [review]
Looks quite OK to me. "No further questions, your honor."

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Karl-WE** [comment]
@illfated  @Kateyanne we have been notified on a different PR that there were issues with the MS queue.
It seems this PR is affected too as the changes are not visible in public days later
https://docs.microsoft.com/en-us/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client

**Comment by illfated** [comment]
Let us hope that this is something being worked on, but if nothing happens within this week, we should definitely ask if we need to redo any of the PRs merged earlier.

I really hope that at least the material already present in the GitHub pages will migrate to docs.microsoft.com when the publishing pipeline issue is resolved.

**Comment by Karl-WE** [comment]
@Kateyanne please review the issue. The merge didn't happen successfully as it appears.
https://docs.microsoft.com/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client
See comment from @eross-msft #4913

**Comment by illfated** [comment]
@Karl-WE : Have you checked the source page to see if your changes have made it there, or if it is possible to remake this PR?

**Comment by Karl-WE** [comment]
hi @illfated I would need to understand more about this works

**Comment by illfated** [comment]
Fair enough. I just thought maybe it would be possible to compare the docs.microsoft.com page
- https://docs.microsoft.com/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client

with the GitHub page
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client.md

and see if the changes from this PR is present in the GitHub page, but not in the docs.microsoft.com version.

**Comment by Karl-WE** [comment]
yes the github version includes this PR. Am I mistaken to relate it to the issue described? Usually merged ones are very soon on the public. What am I mising?

**Comment by illfated** [comment]
That seems to be the issue described earlier and by eross-msft in the other PR. The consequence is that it will not be feasible to create a new PR to push the changes, because the changes are already present in the GitHub source document. We will have to wait for MS Docs technicians to figure out why the document change does not propagate to the docs.microsoft.com site. Thank you for confirming that part. If nothing else, at least we know there is not much we can do from our side.

**Comment by eross-msft** [comment]
@Karl-WE @illfated  I just looked at the live page and I'm seeing the changes here: https://docs.microsoft.com/en-us/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client. Our pipeline is back up and running now, but there are still a few issues to work out.

**Comment by illfated** [comment]
Thank you very much for the update. Much appreciated.

**Comment by Karl-WE** [comment]
Works like a charm! Thank you for the fix Eli @eross-msft !

**Comment by Karl-WE** [comment]
Sorry @eross-msft @illfated got notified by readers on twitter "what changes" and yeah there are gone again. No idea what happens but something is going wild with the docs pages. Changes were visible 4 hours ago (11 pm CET) and gone now (3 am CET)

**Comment by Karl-WE** [comment]
seems like @greg-lindsay merged something and reverted it:
https://github.com/MicrosoftDocs/windows-itpro-docs/commit/2efba68974ecce297716f273393859f0972062ef#diff-2dad0d52195a0bd12948302fe401be600ebb1740c2cba8569fc8b94fb22c74f7

**Comment by illfated** [comment]
Oh. Right. Sorry to hear that. Was the commit described with a reason why the change got reverted, any description for why your changes were considered incorrect?

**Comment by Karl-WE** [comment]
@illfated unfortunately was not able to find a note. I can only assume that @greg-lindsay uses an copy of the repository and due to the backend issues Elizabeth described this caused his copy to be outdated. Wonder why there was no conflict warning. 
In the first place I hope Greg can explain the situation. Hopefully it does not usually happen that later submissions are removing previous changes. Otherwise the work spending on correcting docs would be questionable. Imho this is the good side of git to detect and prevent such issues.

If needed I will do another PR based on your latest version. As quite new external member working on docs, it is important to understand what happens and how to prevent it in the future. 
Thanks for your thoughts and time Greg.

**Comment by illfated** [comment]
Very well, I see. The change has likely taken place in the PR-docs repository, private to the members of the MS Docs team.

**Comment by eross-msft** [comment]
@Karl-WE  @illfated Unfortunately, this isn't my repo, so I don't have admin rights. The best I can do is send @greg-lindsay an email asking him to look at your PR along with his PR and see if he can do anything. It might be that we have to copy the info in this PR, and then create a new PR to add to Greg's content.

**Comment by greg-lindsay** [comment]
@Karl-WE or @illfated if it is simple for you to do so, can you submit a new PR with these changes again so I can merge it?
There might be even more things that need to be redone if all the public changes were lost in the private repo at the end of last week.  I should start by trying to merge the public to private and see if that fixes it

**Comment by greg-lindsay** [comment]
Actually the changes that Karl made are fairly minor I'll just copy them into a new PR and submit it.  But I have sent a note to publishing operations asking how this happened.

**Comment by greg-lindsay** [comment]
Looking at this more carefully, it seems that the only thing that got removed was the text at the top, so the changes were not actually reverted, just apparently some were overwritten. Maybe it was caused by a conflict where the wrong version was accepted for some text.

**Comment by illfated** [comment]
Thank you for the follow-up. Very good. Nice to know.

**Pull request merged**